src/awesome_map/BoundaryTypes.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2014 WebFilings, LLC
```

Comment:
I guess....I just copied and pasted it out of an existing file :stuck_out_tongue: